index.html

```diff
@@ -3,11 +3,64 @@
 <head>
 	<title>JS Onboard Project</title>
 	<script type="text/javascript" charset="utf-8" src="third_party/jquery-2.0.3.min.js"></script>
+	<script type="text/javascript" charset="utf-8" src="app.js"></script>
 </head>
 
-<body>
-	<p>Hello</p>
+<body id="body" style="overflow: hidden; height: 5002px">
+	<div id="form-content">
+		Go To Start Index: <input type="number" id="go-to-index" value="0">
+		<button id="go-to-button" type="button">Go to Index</button>
+		<button id="previous-page" type="button">Go to Previous Page</button>
+		<button id="next-page" type="button">Go to Next Page</button>
+	</div>
+	<table id='WrapperTable' style="width:100%; height:100%; table-layout: fixed;">
```

Comment:
Try keep things constant with formatting. Our convention is to use dashes for `id` and `class` i.e. `id='wrapper-table'`